units/ArmAircraft/armca.lua

```diff
@@ -72,7 +72,6 @@ return {
 		},
 		customparams = {
 			unitgroup = 'builder',
-			area_mex_def = "armmex",
```

Comment:
Is this surely deprecated?

Reply:
Yep, I personally rewrote all the area mex stuff, it uses the buildlist now

Reply:
manual override is still good to have in case the dynamic resolver picks a crappy mex

Reply:
I disagree, hardcoding a bunch of values in specific unitdefs all over the place is far more likely to end up causing errors than a dynamic resolver, and if the dynamic resolver has failure states it will be easier to solve in that isolated code.

Also areamex is primarily used by manually picking the mex type in the UI now, so the "default" choice is rarely relevant anymore.

Reply:
at some point you're going to have mods. hopefully they would have interesting enough mexes to at least consider the possibility the dynamic picker would get it wrong, and they can aren't going to add their own arcane rulesets to the vanilla game

Reply:
to be clear, i don't mind the removal of those specific lines from existing defs, but the removal of the capability to use the override from the gadget